packages/manager/apps/web/client/app/domain/dashboard/domain.service.js

```diff
@@ -1556,14 +1556,7 @@ angular.module('services').service(
         .all(queue)
         .then((results) => {
           const data = results[0] || {};
-          [
-            ,
-            data.dnssec,
-            data.owo,
-            data.owner,
-            data.dns,
-            data.dnsanycast,
-          ] = results;
+          [, data.dnssec, data.owner, data.dns, data.dnsanycast] = results;
```

Comment:
i see that "owo" attribute has been removed from results, is it never used ?

Reply:
no, we are not using it, that's why I removed it.